e-commerce/payment-backend/package.json

```diff
@@ -9,8 +9,8 @@
     "lint-fix": "eslint --fix src"
   },
   "devDependencies": {
-    "@dbos-inc/dbos-cloud": "^1.0.2",
-    "@dbos-inc/dbos-openapi": "^1.0.2",
+    "@dbos-inc/dbos-cloud": "^1.0.3",
```

Comment:
Now there's a version 1.1 to upgrade to! :)

Reply:
Follow-on